test/clj/rems/api/test_forms.clj

```diff
@@ -7,12 +7,14 @@
             [rems.db.api-key :as api-key]
             [rems.db.core :as db]
             [rems.db.test-data-helpers :as test-helpers]
+            [rems.db.testing :refer [owners-fixture +test-api-key+]]
             [ring.mock.request :refer :all])
   (:import (java.util UUID)))
 
 (use-fixtures
   :each
-  api-fixture-without-data)
+  api-fixture-without-data
+  owners-fixture)
```

Comment:
Let's have a look at what the fixture is in the end. Should it be one or several and what the right name and content is.